src/products/products.service.ts

```diff
@@ -30,7 +30,7 @@ export class ProductsService {
     try {
       this.logger.debug(`Updating products table with query "${query}"`);
 
-      return (await this.em.getConnection().execute(query)) as string;
+      return String(await this.em.getConnection().execute(query));
```

Comment:
I see no reason to return query metadata as a reduced string (e.g., [Object object]) by using explicit casting. Please use `void` as the return type.

Reply:
@derevnjuk Instead I have added a 'Success' message when the query executes correctly, as we want to also return the error in case the query fails and I think it's better to keep it consistent

Reply:
@derevnjuk Done, I also added the correct schema for the API